.github/workflows/spelling.yml

```diff
@@ -0,0 +1,19 @@
+name: Spelling
+
+on:
+  pull_request:
+  push:
+
+permissions:
+  contents: read
+
+jobs:
+  typos_check:
+    name: Typos
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
```

Comment:
why needed?

Reply:
It is not needed. Copied from my GHA setup where workflows are stuffed with not needed things.
Deleting ....